torch/csrc/jit/codegen/cuda/executor.cpp

```diff
@@ -422,11 +422,16 @@ FusionExecutor::GlobalBuffers FusionExecutor::allocGlobalVals(
 }
 
 std::vector<at::Tensor> FusionExecutor::allocOutputs(
-    kir::ExpressionEvaluator& expr_eval) {
+    kir::ExpressionEvaluator& expr_eval,
```

Comment:
The skip_io part for aliasing is quite hacky. I think it should work but might get complicated with segmentation.
Need to clean it up later and sort thing straight with segmentation.
Meanwhile, cc'ing @shmsong for a brief review on this.

Reply:
I'm trying to make sure I understand this correctly. So `Fusion::aliasOutputToInput` will be the user/parser facing interface that declares an alias and `skip_io_aliases` assumes the aliased outputs will always be the first few ones declared. Is that correct?

Reply:
IIUC, we are only using aliases for `batchnorm` but how general do we want to fuse it? Essentially this is breaking the SSA of fusion IR and I'd expect quite a few places to need updating. I will try to figure out a plan to make the segmenter support this correctly in edge cases.